packages/react-native/src/breadcrumbs/FileBreadcrumbsStorage.ts

```diff
@@ -32,29 +35,48 @@ export class FileBreadcrumbsStorage implements BreadcrumbsStorage {
     constructor(
         session: SessionFiles,
         private readonly _fileSystem: FileSystem,
-        maximumBreadcrumbs: number,
+        private readonly _limits: BreadcrumbsStorageLimits,
     ) {
         this._sink = new FileChunkSink({
             maxFiles: 2,
             fs: this._fileSystem,
             file: (n) => session.getFileName(FileBreadcrumbsStorage.getFileName(n)),
         });
 
-        this._destinationStream = new WritableStream(
-            new ChunkifierSink({
-                sink: this._sink.getSink(),
-                splitter: () => lineChunkSplitter(Math.ceil(maximumBreadcrumbs / 2)),
-            }),
-        );
+        const splitters: ChunkSplitterFactory<string>[] = [];
+        const maximumBreadcrumbs = this._limits.maximumBreadcrumbs;
+        if (maximumBreadcrumbs !== undefined) {
```

Comment:
double checking: 
maximumBreadcrumbs should always be defined - otherwise, we can generate too big files and cause application problems. I can easily imagine a situation where the customer experiences the issue caused by us overflooding the SD card

Reply:
It's up to the end user to define the valid options. By default, maximum breadcrumbs is 100, so when setting setting this to undefined, you know what you're doing.

You could also say that 10 breadcrumbs could fill up the SD card, if you put a billion bytes in each of them.

Reply:
who wants that? in the end, we're the error-reporting solution, not a full logger. Our main goal is for our library never to crash the app. This is a potential problem.

Reply:
I don't know who wants that. Besides, you could always set the number to something like 2874523758979, and the effect would be the same. You could attach a million attributes to an error, and it could crash the app also.